packages/penrose-core/src/types/types.d.ts

```diff
@@ -56,7 +56,7 @@ type TrMap<T> = { [k: string]: { [k: string]: FieldExpr<T> } };
 interface ITrans<T> {
   // TODO: compGraph
   trMap: TrMap<T>;
-  warnings: string[];
+  warnings: StyError[];
```

Comment:
Does `StyError` exist, or it should be `StyleError`? I saw some renaming earlier